include/avif/avif.h

```diff
@@ -1128,8 +1128,10 @@ AVIF_API avifResult avifEncoderAddImageGrid(avifEncoder * encoder,
                                             avifAddImageFlags addImageFlags);
 AVIF_API avifResult avifEncoderFinish(avifEncoder * encoder, avifRWData * output);
 
-// Codec-specific, optional "advanced" tuning settings, in the form of string key/value pairs.
-// key must be non-NULL, but passing a NULL value will delete that key, if it exists.
+// Codec-specific, optional "advanced" tuning settings, in the form of string key/value pairs,
+// to be sent to codec at the next avifEncoderAddImage() call.
```

Comment:
Nit: add "the" before "codec"

Optional: Perhaps we can say "to be consumed by the codec ..." The word "consume" more strongly implies that the key/value pairs will be cleared after they are sent to the codec.